### PR TITLE
DS-17868 Add acknowledgment in README.md and update CHANGELOG.md

### PR DESCRIPTION
**Pull request by sudiotan**
Changes:
- Add acknowledgment section in cookbook README.md.
- Regenerate CHANGELOG.md to be more readable.

### CODE REVIEW

File: deep-security-agent/README.md [hunk 1]
```diff
@@ -46,4 +46,8 @@ Make sure that you include 'deep-security-agent' in your node's 'run_list'. This
     "recipe[deep-security-agent::default]"
   ]
 }
-```
\ No newline at end of file
+```
+
+## Acknowledgment
+
+* We thank [Blair Hamilton](https://github.com/blairham) for his contributions and put this cookbook into [Supermarket](https://supermarket.chef.io/cookbooks/deep-security-agent).
```
Comment (ChristinaSmith123) [inline deep-security-agent/README.md]: change:
'for his contributions and put this cookbook into Supermarket' -> 'for his contributions to the [Deep Security Agent cookbook] (https://supermarket.chef.io/cookbooks/deep-security-agent).'

Comment (ChristinaSmith123) [inline deep-security-agent/README.md]: change:
'for his contributions and put this cookbook into Supermarket' -> 'for his contributions to the [Deep Security Agent cookbook] (https://supermarket.chef.io/cookbooks/deep-security-agent).'

**Review by ChristinaSmith123 — CHANGES_REQUESTED** [review]
Please change acknowledgment of Blair Hamilton as indicated.

File: deep-security-agent/CHANGELOG.md [hunk 2]
```diff
@@ -1,43 +1,17 @@
...
-fea5c69t(7 months ago): Updated README
-baaf4dat(7 months ago): Updated README
-8e8c5c8t(7 months ago): Update README with the required attributes for the deploy-dsa.rb recipe
-eda4a3et(7 months ago): Removed port from dsm_agent_download_hostname, it's in it's own attribute now
-960935bt(7 months ago): Added initial test recipe for deploying the Deep Security agent
-11118e7t(7 months ago): Added a default cookbook
-cd07e1bt(7 months ago): Updated README
+# Change Log
+
+## [1.1.0](https://github.com/deep-security/chef/tree/1.1.0) (2018-01-17)
```
Comment (DeepSecurityOpenSourceSupport) [inline deep-security-agent/CHANGELOG.md]: As the change log will not contain the release itself, the in consist may cause confusing.
I suggest to show the release page link on github directly
https://github.com/deep-security/chef/releases

**Pull request merged**